server/src/main/java/org/opensearch/index/store/Store.java

```diff
@@ -1394,9 +1385,9 @@ public void seek(long pos) throws IOException {
                     // skipping the verified portion
                     input.seek(verifiedPosition);
                     // and checking unverified
-                    skipBytes(pos - verifiedPosition);
+                    super.seek(pos);
```

Comment:
I don't understand this change, could you clarify please? Seems like we trading `skipBytes` to `skipByReading` and I don't understand why (sorry)

Reply:
It's related to [LUCENE-9794](https://issues.apache.org/jira/browse/LUCENE-9794) and 

```
  /* This buffer is used when skipping bytes in skipBytes(). Skipping bytes
   * still requires reading in the bytes we skip in order to update the checksum.
   * The reason we need to use an instance member instead of sharing a single
   * static instance across threads is that multiple instances invoking skipBytes()
   * concurrently on different threads can clobber the contents of a shared buffer,
   * corrupting the checksum. See LUCENE-5583 for additional context.
   */

Reply:
:+1: thanks a lot